blog/2024-07-19-coordinator_async_setup.md

```diff
@@ -0,0 +1,43 @@
+---
+author: Josef Zweck
+authorURL: https://github.com/zweckj
+title: Setting up `DataUpdateCoordinators` with `_async_setup`
+---
+
+Until now, there was no clearly defined place where you would initialize your `DataUpdateCoordinator`,
+or where you could load data, that only needs to be loaded once.
```

Comment:
the constructor initializes the data update coordinator. what we now added is a way to prepare before the coordinator starts